### PR TITLE
Document using Prometheus metrics

### LINKED ISSUE #1307: Add docs for Prometheus metrics
We don't seem to have any documentation about the Prometheus metrics endpoint for Docker (new in 1.13) yet.

There is a daemon option `--metrics-addr` to configure (currently experimental I think), but it is really only experimental in that the exact metrics may change, this feature is not going away and will become increasingly important.

(not sure if best place - I couldnt find anything anywhere)
File: [engine/admin/runmetrics.md](https://docs.docker.com/engine/admin/runmetrics/)

The only thing on the blog is https://blog.docker.com/2016/12/docker-prometheus-joint-holiday-meetup-recap/

The 1.13 blog briefly mentions it and links to the PR https://github.com/docker/docker/pull/25820

cc @stevvooe @thaJeztah

### PR DESCRIPTION
**Pull request by mstanleyjones**
### Proposed changes

Basic setup instructions for using Prometheus with Docker

Fixes #1307 

@stevvooe @crosbymichael PTAL and ping anyone else who can help fill in the blanks

### CODE REVIEW

File: engine/admin/prometheus.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    # Load rules once and periodically evaluate them according to the global 'evaluation_interval'.
+    rule_files:
+      # - "first.rules"
+      # - "second.rules"
+
+    # A scrape configuration containing exactly one endpoint to scrape:
+    # Here it's Prometheus itself.
+    scrape_configs:
+      # The job name is added as a label `job=<job_name>` to any timeseries scraped from this config.
+      - job_name: 'prometheus'
```
Comment (stevvooe) [inline engine/admin/prometheus.md]: You should create a new job for `docker`. This job is for monitoring prometheus itself. Just add another section like this:

```
- job_name: 'docker'
         # metrics_path defaults to '/metrics'
         # scheme defaults to 'http'.
 
         static_configs:
           - targets: ['localhost:9323']
```

File: engine/admin/prometheus.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+2.  Start a Prometheus container using this configuration.
+
+    ```bash
+    $ docker run -p 9090:9090 \
+      -v /tmp/prometheus.yml:/etc/prometheus/prometheus.yml \
+      --name my-prometheus \
+      prom/prometheus
+    ```
+
+    TBD: use a service instead?
```
Comment (stevvooe) [inline engine/admin/prometheus.md]: I don't remember how to get this working...

File: engine/admin/prometheus.md (hunk 1 above)
Comment (mstanleyjones) [inline engine/admin/prometheus.md]: I updated the config to do it this way now.

**Comment by mstanleyjones** [comment]
I tried using `--net host` and the situation is even worse now. Prometheus is probably working fine but i can't connect to its web interface, presumably because it is only available from inside the Moby VM.

**Comment by mstanleyjones** [comment]
Added some graphics and tabs for Linux and Mac/Windows.

**Comment by mstanleyjones** [comment]
Updated to use Prometheus as a service, so we don't have to work so hard to get it working on D4M!

**Pull request merged**